tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "noImplicitReturns": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
+    "importHelpers": true,
```

Comment:
tslib is dep of bundler, bundler should do that for you.
`importHelpers` is overriden to `false` for now in bunchee. I guess I need to change a few behavior to let user customize it..